typings/index.d.ts

```diff
@@ -3028,6 +3028,11 @@ declare module 'discord.js' {
     around?: Snowflake;
   }
 
+  type ChannelMention = `<#${Snowflake}>`;
+  type UserMention = `<@${Snowflake}>`;
+  type MemberMention = UserMention | `<@!${Snowflake}>`;
+  type RoleMention = '@everyone' | `<@&${Snowflake}>`;
```

Comment:
I thought these were meant to be sorted alphabetically but at least leave an empty line between them

Reply:
I can sort them alphabetically but I didn't add a line because I saw some other types without it... I'll check this later.

Reply:
Then those should probably have it cuz there's an empty line between almost everything in this file. And yeah you should sort them alphabetically within the typedefs region